detail.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="app_base.htm" />
+
+
+<%block name="content">
+
+<h1 class="text-center">${ product.name | h }</h1>
+<div class="product_detail_flexbox">
+    <ul class="product_detail_list">
+        <%
+        images = product.image_urls()
+        %>
+        %for img in images:
```

Comment:
You could just include this variable declaration in the for loop, instead of creating a new variable.